From 7cf041f242b475b6bea0dc48dd2a895727efe6cc Mon Sep 17 00:00:00 2001
From: Pino Toscano <ptoscano@redhat.com>
Date: Fri, 31 Jan 2014 13:16:39 +0100
Subject: [PATCH 033/102] sysprep: adapt test-virt-sysprep-passwords.sh to
 libguestfs 1.20

a) do not use $guestfish, but call "../fish/guestfish" directly
b) use "get-attach-method" instead of "get-backend"
---
 sysprep/test-virt-sysprep-passwords.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sysprep/test-virt-sysprep-passwords.sh b/sysprep/test-virt-sysprep-passwords.sh
index 9280fc5..565af5b 100755
--- a/sysprep/test-virt-sysprep-passwords.sh
+++ b/sysprep/test-virt-sysprep-passwords.sh
@@ -21,7 +21,7 @@ set -e
 
 # Test all the combinations of password options.
 
-if [ "$($guestfish get-backend)" = "uml" ]; then
+if [ "$(../fish/guestfish get-attach-method)" = "uml" ]; then
     echo "$0: skipping test because uml backend does not support qcow2"
     exit 77
 fi
-- 
1.9.3

